webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py

```diff
@@ -38,44 +38,73 @@
 
 # pylint: disable=too-many-instance-attributes
 class ReservoirSimulationTimeSeriesRegional(WebvizPluginABC):
-    """### ReservoirSimulationTimeSeriesRegional
-
-This plugins aggregates and visualizes regional time series data from simulation ensembles. That
-is cumulatives, rates and inplace volumes. In addiotion recovery is calculated based on the
-inplace volumes.
-
-#### Mandatory input
-* `ensembles`: Which ensembles in `shared_settings` to include in the plugin.
-
-#### Optional input
-* `fipfile`: Path to a yaml-file that defines a match between FIPXXX (e.g. FIPNUM) regions
- and _'human readable'_ regions, zones and etc. These will be used as filters. _Note that this is a
- single file used for all ensembles_. If all region numbers for a filter are missing in the data,
- this filter value will be silently ignored. E.g. if no vectors matches 5 or 6 in
- [this example file](
- https://github.com/equinor/webviz-subsurface-testdata/tree/master/reek_history_match/share/
-regions/fip.yaml), `ZONE` == `LowerReek` would be ignored in the plugin for `FIPNUM`. This is
- to allow you to use the same file for e.g. a sector and a full field model.
-* `initial_vector`: First vector to plot (default is `ROIP`)
-* `column_keys`: List of vector patterns to include or `None`, the latter gives all available
- vectors, and `None` is also the default.<br/>
+    """Aggregates and visualizaes regional time series data from simulation ensembles. That
+is: cumulatives, rates and inplace volumes. Allows human friendly filter names, e.g. regions,
+zones and etc based on user input.
+
+In addition recovery is calculated based on the changes in aggregated inplace volumes, \
+as long as all historical data is present in the data.
+
+Example of aggregation of ROIP over regions in filter: \
+$Agg(ROIP)_{date} = \\sum_{N\\in filter} {ROIP_N}_{date}$
```

Comment:
```suggestion
${\\rm Agg}({\\rm ROIP})_{\\rm date}} = \\sum_{N\\in \\rm filter}} {\\rm ROIP}_N}_{\\rm date}}$
```
Only a style suggestion to consider comitting to.... I'm used to roman style on variables that are spelled out with "text".

Reply:
In that case I would suggest to keep it clean, drop roman and go sans serif (\\sf) as the rest of the text. Comparison of 
sans serif (upper equation) and roman (lower equation):
![image](https://user-images.githubusercontent.com/47146384/87160661-9b694480-c2c3-11ea-9c07-4057de1596fe.png)